### PR TITLE
Register and Tag NetworkLayout: v0.0.1

### PR DESCRIPTION
**Pull request by abhijithanilkumar**
[NetworkLayout.jl](https://github.com/JuliaGraphs/NetworkLayout.jl) is a package that houses graph and tree layout algorithms in Julia. Uses algorithms from GraphLayout, GraphPlot and PlotRecipes.

### CODE REVIEW

**Comment by tbreloff** [comment]
Congrats on the package. Looking forward to incorporating this into
PlotRecipes.

**Comment by tkelman** [comment]
Please push the git tag so it shows up under https://github.com/JuliaGraphs/NetworkLayout.jl/releases

**Comment by tkelman** [comment]
`src/stress.jl` uses FixedSizeArrays but your REQUIRE file does not declare that dependency. You shouldn't do `Pkg.clone("LightGraphs")` on travis, let `test/REQUIRE` handle that.

**Comment by abhijithanilkumar** [comment]
@tkelman The release version of LightGraphs throws errors with 0.5. That was why `Pkg.clone("LightGraphs")` was included.

**Comment by tkelman** [comment]
>  I've added FixedSizeArrays as dependency

You added it to `test/REQUIRE`. That's not right for a dependency that you use in `src`, it needs to be in `REQUIRE`

**Comment by abhijithanilkumar** [comment]
> You added it to test/REQUIRE. That's not right for a dependency that you use in src, it needs to be in REQUIRE

Sorry. My bad. Corrected it.

**Comment by tkelman** [comment]
the requirement needs to be reflected in the requires files of the metadata tags too.

**Comment by abhijithanilkumar** [comment]
Made the changes.

File: NetworkLayout/versions/0.0.2/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+20a54d9a28d1aff7b471b9f99e2e73f16a7b2db3
```
Comment (tkelman) [inline NetworkLayout/versions/0.0.2/sha1]: these sha's are inconsistent with your git tags https://github.com/JuliaGraphs/NetworkLayout.jl/releases

**Comment by abhijithanilkumar** [comment]
@tkelman Is it okay now? I've reset the tags.

**Pull request merged**